src/routes/auth/+page.svelte

```diff
@@ -3,12 +3,14 @@
 	import i18n from '$lib/i18n';
 	import { addToast } from '$lib/toaster.svelte';
 	import { goto } from '$app/navigation';
+	import { page } from '$app/stores';
 
 	const { data } = $props();
 	const { supabase } = data;
 
 	let email = $state('');
 	let password = $state('');
+	const redirect = $derived($page.url.searchParams.get('redirect') || '/');
```

Comment:
```suggestion
	const redirect = $derived($page.url.searchParams.get('redirect') || '/dashboard');
```
On veut pas rediriger sur le dashboard après un login ?

Reply:
Il y'a un cas : si tu es juste `admin_level === 'user'` et que tu veux accéder au frontend. Pour ce cas je pense le laisser

Reply:
Okay